jengarootReal.re

```diff
@@ -70,7 +75,11 @@ let ocamlDep sourcePath::sourcePath => {
         fun () =>
           bashf
             dir::srcDir
-            "ocamldep -pp refmt -ml-synonym .re -mli-synonym .rei -modules -one-line %s %s"
+            (
+              isReasonSrc sourcePath ?
+                "ocamldep -pp refmt -ml-synonym .re -mli-synonym .rei -modules -one-line %s %s" :
+                "ocamldep -ml-synonym .ml -mli-synonym .mli -modules -one-line %s %s"
```

Comment:
is this necessary? Those are the defaults. Also what happens when you have mixed ml and re files?